src/config.rs

```diff
@@ -35,6 +42,15 @@ pub struct Config {
     pub token: Option<String>,
 }
 
+pub fn merge_configs(first: PdbConfigSection, second: PdbConfigSection) -> PdbConfigSection {
```

Comment:
I think there should be some tests of this merge and the overlay from the global config section